### PR TITLE
Home page tiles

### LINKED ISSUE #39: Home Page Tiles
Change the home page layout from a list to grid.

Each tile should have an image, stream title, and possibly tag.

### PR DESCRIPTION
**Pull request by jeffh1996**
Screenshot (3)

### CODE REVIEW

**Comment by jeffh1996** [comment]
#39

File: client/src/components/streams/StreamList.js [hunk 1]
```diff
@@ -11,52 +11,58 @@ class StreamList extends React.Component {
...
+    var src = thumbnails["default"];
+    let iconWidth = "40%";
+    let iconHeight = "60%";
     if (stream.gametag !== undefined) {
       for (var key in thumbnails) {
         if (key === stream.gametag.toLowerCase()) {
           src = thumbnails[key];
+          iconWidth = "100%";
         }
+        console.log(key)
```
Comment (emilio-alvarez) [inline client/src/components/streams/StreamList.js]: Did you forget to remove this?

File: client/src/components/streams/StreamList.js [hunk 2]
```diff
@@ -11,52 +11,58 @@ class StreamList extends React.Component {
...
-            <div className="content">
+            <div class="center aligned" style={{width:"100%",overflow:"hidden",maxHeight:"3em"}}>
               <Link to={`/streams/${stream.userid}`} className="header">
                 {stream.title}
               </Link>
-              <div className="description white-text">
-                {stream.description}
-              </div>
             </div>
+            <div class="ui fragment description white-text" style={{width:"100%",overflow:"hidden",height:"4em"}}>{stream.description}</div>
```
Comment (emilio-alvarez) [inline client/src/components/streams/StreamList.js]: Up to you, but if you add content-scrollable as a class and remove "overflow:"hidden"" from this it lets you scroll the description.

**Review by emilio-alvarez — COMMENTED** [review]
Looks good and is a bit more responsive. Were you unable to figure out how to make clicking the div for each stream navigate to the stream? I may be wrong, but I think you might be able to do this:
1) import history from "../../history"
2) add an onclick handler to the div you want to make clickable with onClick={() => this.clickHandler(stream}}
3) In the handler, do history.push(\`/streams/${stream.userid}\`)

Whether you would want to make the whole div clickable, or just the picture, or whatever, is up to you.

**Review by emilio-alvarez — APPROVED** [review]
Nice!

**Pull request merged**